test/acceptance/objects/auto_schema_test.go

```diff
@@ -97,13 +97,13 @@ func autoSchemaObjects(t *testing.T) {
 							"longitude": json.Number("1.01"),
 						},
 						"testPhoneNumber": map[string]interface{}{
-							"input":                  "020 1234567",
-							"internationalFormatted": "+31 20 1234567",
-							"defaultCountry":         "nl",
-							"countryCode":            31,
-							"national":               201234567,
-							"nationalFormatted":      "020 1234567",
-							"valid":                  true,
+							"input": "020 1234567",
+							// "internationalFormatted": "+31 20 1234567",
+							"defaultCountry": "nl",
+							// "countryCode":            31,
+							// "national":               201234567,
+							// "nationalFormatted":      "020 1234567",
+							// "valid":                  true,
```

Comment:
should this be left commented? or can we uncomment this?

Reply:
removed, as those are read only fields